src/Infrastructure/Symfony/Bundle/DependencyInjection/Compiler/CallableFirstParameterExtractor.php

```diff
@@ -2,8 +2,11 @@
 
 namespace CodelyTv\Infrastructure\Symfony\Bundle\DependencyInjection\Compiler;
 
+use CodelyTv\Shared\Domain\Bus\Event\DomainEventSubscriber;
 use ReflectionClass;
 use ReflectionMethod;
+use function Lambdish\Phunctional\reduce;
+use function Lambdish\Phunctional\reindex;
 
 final class CallableFirstParameterExtractor
```

Comment:
It would be nice to add a class DocBlock in this case explaining a little bit its use cases 🙂